.eslintrc.json

```diff
@@ -30,6 +30,7 @@
     "no-case-declarations": 1,
     "no-confusing-arrow": 1,
     "no-console": 1,
+    "no-debugger": 1,
```

Comment:
I didn't know `debugger` was a thing in JavaScript :-o

Reply:
:grin: I like it